acceptance/github_asset_fetcher.go

```diff
@@ -196,14 +196,25 @@ func (f *GithubAssetFetcher) FetchReleaseVersion(owner, repo string, n int) (str
 	}
 
 	// get all release versions
-	releases, _, err := f.githubClient.Repositories.ListReleases(f.ctx, owner, repo, nil)
+	rawReleases, _, err := f.githubClient.Repositories.ListReleases(f.ctx, owner, repo, nil)
 	if err != nil {
 		return "", errors.Wrap(err, "listing releases")
 	}
-	if len(releases) == 0 {
+	if len(rawReleases) == 0 {
 		return "", fmt.Errorf("no releases found for %s/%s", owner, repo)
 	}
 
+	// exclude drafts
+	var releases []*github.RepositoryRelease
+	for _, release := range rawReleases {
+		if !*release.Draft {
```

Comment:
Would this be better written as `release.Draft != nil` if that's the same thing?

I'm having trouble mapping `!*`.

Reply:
@natalieparellano To unblock a PR, I pushed that change to this branch.

Reply:
@dfreilich thanks for attempting the change. I saw that the change didn't yield the results that we wanted. I'm taking it in as is. We can decipher it later. :D